app/views/collection/show.html.slim

```diff
@@ -80,8 +80,12 @@
       =label_tag 'search', "Search for works", class: 'hidden'
         
     -if @collection.active?
-      =link_to "Pages That Need Transcription", collection_needs_transcription_path(@collection.owner, @collection), class: 'button review-button'
-
+      -if @collection.works.where(:ocr_correction => true).present? && @collection.works.where(:ocr_correction => false).present?
+        =link_to "Pages That Need Correction or Transcription", collection_needs_transcription_path(@collection.owner, @collection), class: 'button review-button'
```

Comment:
Can we externalize these messages?  We'll need a way to mark them in the dictionary file as "needing translation" -- maybe with a comment that the translator can remove once it's translated?